src/graphics/renderer.h

```diff
@@ -119,6 +123,7 @@ class Renderer {
 
 	glm::mat4 _view;
 	glm::mat4 _projection;
+	glm::vec2 _renderPlane;
```

Comment:
I would prefer a name like "_viewportSize". Additionally, you could use glm::size2 instead of glm::vec2.

Reply:
Although `glm::size2` does make sense here since `_viewportSize` is not meant to have dimensions with a fractional part, many places throughout the codebase require passing its members as a float. Thus, storing them as an int would necessitate some extra int-to-float conversions, hence why I am sticking with `glm::vec2` for now.